libs/application/templates/mortgage-certificate/src/fields/SelectProperty/PropertyTypeSearchField/PropertyTable/RealEstateTable.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { FC } from 'react'
+import { Table as T, Checkbox } from '@island.is/island-ui/core'
+import { FieldBaseProps } from '@island.is/application/types'
+import { RealEstateDetail } from '@island.is/api/schema'
+import { useLocale } from '@island.is/localization'
+import { FieldArrayWithId } from 'react-hook-form'
+import { MortgageCertificate } from '../../../../lib/dataSchema'
+import { propertySearch } from '../../../../lib/messages'
+import { SelectedProperty } from '../../../../shared'
+import { TableHeadText } from './TableHeadText'
+
+interface RealEstateTableProps {
+  selectHandler: (property: SelectedProperty, index: number) => void
+  realEstate: RealEstateDetail[] | undefined
+  checkedProperties: FieldArrayWithId<
+    MortgageCertificate,
+    'selectedProperties.properties',
+    'id'
+  >[]
+}
+
+export const RealEstateTable: FC<
```

Comment:
Have you checked if you can use the `buildTableRepeaterField `or  `buildStaticTableField`